tests/test_bot.py

```
@@ -0,0 +1,931 @@
+    ):
+        """Test main function with access token from environment."""
+        # Setup mocks - ensure credentials are found
+        mock_load_config.return_value = sample_config
+        mock_load_env.return_value = (
+            "test_access_token",  # Provide access token instead of relying on credentials
+            {"esv": "test_key"},
+        )
+
+        mock_load_creds.return_value = None  # No saved credentials
+
+        mock_get_store.return_value = Path("/tmp/store")
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/store"

(S108)